merlin/models/tf/inputs/base.py

```diff
@@ -229,7 +226,7 @@ def InputBlockV2(
         Schema of the input data. This Schema object will be automatically generated using
         [NVTabular](https://nvidia-merlin.github.io/NVTabular/main/Introduction.html).
         Next to this, it's also possible to construct it manually.
-    embeddings : Optional[Block], optional
+    categorical : Optional[Block], optional
```

Comment:
docstring needs updating to match new arguments and types:
- new arg `continuous`
- typehint here for `categorical` arg. 
- removal of params `continuous_projection` and `continuous_column_selector`